framework/test/jest/unit/specs/controller/application_state.spec.js

```diff
@@ -196,7 +196,7 @@ describe('Application State', () => {
 
 			it('should call get four times', async () => {
 				// Assert
-				expect(spies.get).toHaveBeenCalledTimes(4);
+				expect(spies.get).toHaveBeenCalledTimes(5);
```

Comment:
why this increase?

Reply:
Removed the test case as it is difficult to maintain and will be refactored in #3875